docs/agents/sematext-agent/linux-installation.md

```diff
@@ -0,0 +1,305 @@
+title: Installing Sematext Agent on Linux
+
+The Sematext Agent is available as a package (DEB, RPM, etc.) called `sematext-agent`:
+
+<div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect">
+ <div class="mdl-tabs__tab-bar">
+     <a href="#redhat" class="mdl-tabs__tab is-active">Red Hat</a>
+     <a href="#centos" class="mdl-tabs__tab">CentOS</a>
+     <a href="#debian" class="mdl-tabs__tab">Debian</a>
+     <a href="#ubuntu" class="mdl-tabs__tab">Ubuntu</a>
+     <a href="#amazonlinux" class="mdl-tabs__tab">Amazon</a>
+     <a href="#fedora" class="mdl-tabs__tab">Fedora</a>
+     <a href="#suse" class="mdl-tabs__tab">SuSE</a>
+ </div>
+
+ <div class="mdl-tabs__panel is-active" id="redhat">
+   <h3>Installing on Red Hat</h3>
+   <pre>
+sudo wget https://pub-repo.sematext.com/redhat/sematext.repo -O /etc/yum.repos.d/sematext.repo
+sudo yum clean all
+sudo yum install sematext-agent
+   </pre>
+
+   <h3>Upgrading/Updating on Red Hat</h3>
+   1. Update the repository file and GPG key:
+   <pre>
+sudo wget https://pub-repo.sematext.com/redhat/sematext.repo -O /etc/yum.repos.d/sematext.repo
+sudo rpm --import https://pub-repo.sematext.com/redhat/sematext.gpg.key
+   </pre>
+   2. Upgrade the Sematext Agent:
+   <pre>
+sudo yum update sematext-agent
+   </pre>
+
+   <h3>Uninstalling on Red Hat</h3>
+   <pre>
+sudo yum remove sematext-agent
+   </pre>
+   To ensure all configuration files and logs are removed, delete the main installation directory:
+   <pre>
+sudo rm -rf /opt/spm
+   </pre>
+ </div>
+
+ <div class="mdl-tabs__panel" id="centos">
+   <h3>Installing on CentOS</h3>
+   <pre>
+sudo wget https://pub-repo.sematext.com/centos/sematext.repo -O /etc/yum.repos.d/sematext.repo
+sudo yum clean all
+sudo yum install sematext-agent
+   </pre>
+
+   <h3>Upgrading/Updating on CentOS</h3>
+   1. Update the repository file and GPG key:
+   <pre>
+sudo wget https://pub-repo.sematext.com/centos/sematext.repo -O /etc/yum.repos.d/sematext.repo
+sudo rpm --import https://pub-repo.sematext.com/centos/sematext.gpg.key
+   </pre>
+   2. Upgrade the Sematext Agent:
+   <pre>
+sudo yum update sematext-agent
+   </pre>
+
+   <h3>Uninstalling on CentOS</h3>
+   <pre>
+sudo yum remove sematext-agent
+   </pre>
+   To ensure all configuration files and logs are removed, delete the main installation directory:
```

Comment:
>   To ensure all configuration files and logs are removed, delete the main installation directory:

Could you please add a similar line to Windows page.
They need to remove ProgramFiles/STA folder and also logs/journal folders.

Removing the package won't be deleting configs starting from 3.7.0. Also logs/journal folders are not deleted today.

Reply:
C:\Windows\System32\config\systemprofile\AppData\Local\sematext-agent

Reply:
C:\Program Files\Sematext Agent

for properties/ and some small files